src/objective-c/change-comments.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python
```

Comment:
This should probably be `#!/usr/bin/env python2.7`, as is done elsewhere in the codebase. (Do you see any place where we have `#!/usr/bin/python` in the codebase today?)